UnderSea.Angular/src/app/core/pages/login/login.component.ts

```diff
@@ -46,32 +46,33 @@ export class LoginComponent implements OnInit {
   }
 
   login(): void {
-
-    this.authService.login(this.username, this.password).pipe(
-      tap(tokens => {
+    this.authService.login(
+      this.loginForm.controls.username.value,
+      this.loginForm.controls.password.value)
+      .subscribe(tokens => {
```

Comment:
Ez maradjon pipe-al megoldva, az jó, a loginformosat én is megcsináltam egyébként, valamiért az előző loginos merge-nél elveszett ez

Reply:
Okés, visszaraktam pipe-ba